tools/setup/install-sgrep

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+
+if [ ! -d "/usr/local/bin/sgrep-lint-files" ]
+then
+    tmpdir="$(mktemp -d)"
+    trap 'rm -r "$tmpdir"' EXIT
+    cd "$tmpdir"
+    wget -nv "https://github.com/returntocorp/sgrep/releases/download/v0.4.9b2-ubuntu-18.04/sgrep-0.4.9b2-ubuntu-18.04.tgz"
+    tar -xzf "sgrep-0.4.9b2-ubuntu-18.04.tgz" -C /usr/local/bin/
+    ln -sf /usr/local/bin/sgrep-lint-files/sgrep-lint /usr/local/bin/sgrep-lint
+    ln -sf /usr/local/bin/sgrep-lint-files/sgrep /usr/local/bin/sgrep
+fi
```

Comment:
We should fix the missing newline at EOF here.

It makes me a little sad to have a custom tarball-based install script like this, but I guess `sgrep` isn't available via pypi/npm, so what's there to do?  But possibly you folks should look at publishing in `npm` or something so we don't need a special script for just it.

In any case, we should definitely have this check the hash of the tarball before installing.